src/solutions/save_idxs.jl

```diff
@@ -105,6 +105,20 @@ struct SavedSubsystem{V, T, M, I, P, Q, C}
     partition_count::C
 end
 
+SavedSubsystem(indp, pobj, ::Nothing) = nothing
+
+function SavedSubsystem(indp, pobj, saved_idxs::Vector{Int})
+    isempty(saved_idxs) && return nothing
+    isempty(variable_symbols(indp)) && return nothing
```

Comment:
won't this cause inference issues?

Reply:
I think the `isempty(saved_idxs)` case can be removed. I'll try and see if I can figure out the second case from type information.